Articles component

Ebenezr opened the pull request:
![image](https://user-images.githubusercontent.com/37300065/202200465-6a3f62b2-29a6-449b-87b6-dc48b4b4b878.png)

nyandika commented on src/components/Articles/BlogCards.tsx:
```diff
@@ -0,0 +1,126 @@
+  videoUrl?: string;
+}
+
+const BlogCards: React.FC = () => {
```
This is not a recommended way of creating components. it's been removed from template actually:  [Link](https://github.com/facebook/create-react-app/pull/8177)

nyandika replied:
you can use: `const Blogcard = () => {}`

nyandika commented on src/components/Articles/BlogCards.tsx:
```diff
@@ -0,0 +1,126 @@
+  };
+  // onmount hydrate articles state
+  useEffect(() => {
+    getArticles().then((response) => setArticles(response?.data));
```
You can handle all fetching and parsing in the `getArticles` method and just get the data here

nyandika replied:
Yea this is perfect

nyandika reviewed (changes requested):
You should also commit the package-lock file

I also noticed you deleted the other PR to create a new one, you dont have to do this. making changes on the same branch and pushing to the same branch will automatically update the PR. let me know if you need further help on that and we can setup a small sync

Ebenezr commented:
noted

nyandika reviewed (approved):
There are a couple of nits that we could refactor as we go but this is good progress

nyandika commented:
I pushed some updates to lib dependencies which had led to conflicts on this branch. Let me know if you need help resolving this. I can set up a sync

Ebenezr commented:
![image](https://user-images.githubusercontent.com/37300065/204517529-3691c46a-bb92-48fc-85fb-1321e67e24d3.png)

I have fixed the conflicts, but I'm version error when I try to npm install, 
NB: I have updated my node and npm to the latest version already.
